sdk/python/foundation-models/system/inference/text-to-image/safe-image-text-to-image-online-endpoint.ipynb

```diff
@@ -0,0 +1,761 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create an Azure AI Content Safety __(AACS)__ enabled Image to Image online endpoint (Preview)\n",
+    "### This notebook is under preview.\n",
+    "\n",
+    "### Steps to create an __AACS__ enabled __image-to-image__ online endpoint\n",
+    "1. Create an __AACS__ resource for moderating the request from user and response from the __image-to-image__ online endpoint.\n",
+    "2. Create a new __AACS__ enabled __image-to-image__ online endpoint with a custom [score_online.py](./aacs-scoring-files/score/score_online.py) which will integrate with the __AACS__ resource to moderate the response from the __image-text-to-image__ model and the request from the user, but to make the custom [score_online.py](./aacs-scoring-files/score/score_online.py) to successfully authenticated to the __AACS__ resource, we have 2 options:\n",
+    "    1. __UAI__, recommended but more complex approach, is to create a User Assigned Identity (UAI) and assign appropriate roles to the UAI. Then, the custom [score_online.py](./aacs-scoring-files/score/score_online.py) can obtain the access token of the UAI from the AAD server to access the AACS resource. Use [this notebook](aacs-prepare-uai.ipynb) to create UAI account for step 3 below\n",
+    "    2. __Environment variable__, simpler but less secure approach, is to just pass the access key of the AACS resource to the custom [score_online.py](./aacs-scoring-files/score/score_online.py) via environment variable, then the custom [score_online.py](./aacs-scoring-files/score/score_online.py) can use the key directly to access the AACS resource, this option is less secure than the first option, if someone in your org has access to the endpoint, he/she can get the access key from the environment variable and use it to access the AACS resource.\n",
+    "\n",
+    "\n",
+    "### Task\n",
+    "`image to image` task takes an original image, a text prompt as input. The model generates an image by modifying the original image.\n",
+    "\n",
+    " \n",
+    "### Model\n",
+    "Models that can perform the `image-to-image` task are tagged with `image-text-to-image`. We will use the `stabilityai-stable-diffusion-xl-refiner-1-0` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name.\n",
```

Comment:
Task name. Same comment as above.

The task should be image-to-image.
Have left the same comment on model asset PR - https://github.com/Azure/azureml-assets/pull/1785.